src/vs/workbench/parts/search/browser/openFileHandler.ts

```diff
@@ -118,25 +118,26 @@ export class OpenFileHandler extends QuickOpenHandler {
 		return promise.then(result => [new QuickOpenModel(result[0]), result[1]]);
 	}
 
-	private doFindResults(searchValue: string): TPromise<[QuickOpenEntry[], ISearchStats]> {
+	private doFindResults(searchValue: string): PPromise<[QuickOpenEntry[], ISearchStats], QuickOpenEntry> {
 		const query: IQueryOptions = {
 			folderResources: this.contextService.getWorkspace() ? [this.contextService.getWorkspace().resource] : [],
 			extraFileResources: getOutOfWorkspaceEditorResources(this.editorGroupService, this.contextService),
 			filePattern: searchValue
 		};
 
+		let results: QuickOpenEntry[] = [];
 		return this.searchService.search(this.queryBuilder.file(query)).then((complete) => {
-			let results: QuickOpenEntry[] = [];
```

Comment:
According to coding standards, `.then(complete => [results, complete.stats], null, progress => { ...});` would be better